### PR TITLE
Mustache down

### LINKED ISSUE #28: mustachify.me DOWN
http://mustachify.me/ has been down å long time, please remove the mustache me function

### PR DESCRIPTION
**Pull request by HaroldPutman**
Fix #28 by showing warning when custom mustachify server is not set up.

Sorry, the Mustachify server is not configured.
no mustache

### CODE REVIEW

**Comment by davefp** [comment]
:+1: :shipit:

File: src/google-images.coffee [hunk 1]
```diff
@@ -35,9 +34,12 @@ module.exports = (robot) ->
         msg.send url
 
   robot.respond /(?:mo?u)?sta(?:s|c)h(?:e|ify)?(?: me)? (.+)/i, (msg) ->
+    if not process.env.HUBOT_MUSTACHIFY_URL?
+      msg.send "Sorry, the Mustachify server is not configured."
```
Comment (technicalpickles) [inline src/google-images.coffee]: If you do a single `msg.send` with multiple arguments, that should help ensure the order of messages is consistent.

File: src/google-images.coffee [hunk 2]
```diff
@@ -35,9 +34,12 @@ module.exports = (robot) ->
...
+      msg.send "http://i.imgur.com/BXbGJ1N.png"
+      return
     mustacheBaseUrl =
```
Comment (technicalpickles) [inline src/google-images.coffee]: This can probably be a single line now.

**Comment by technicalpickles** [comment]
Left a few small things, but otherwise looks good :+1:

File: src/google-images.coffee (hunk 2 above)
Comment (HaroldPutman) [inline src/google-images.coffee]: Thanks for the tip about the message order!
One line seems unreadable to me
```
return msg.send "Sorry, the Mustachify server is not configured.", "http://i.imgur.com/BXbGJ1N.png" if not process.env.HUBOT_MUSTACHIFY_URL?
```
So how about
```
    if not process.env.HUBOT_MUSTACHIFY_URL?
      msg.send "Sorry, the Mustachify server is not configured."
        , "http://i.imgur.com/BXbGJ1N.png"
      return
```

Comment (technicalpickles) [inline src/google-images.coffee]: Sure, multiple arguments over multiple lines is totally fine.

**Pull request merged**